src/isIsogram.test.js

```diff
@@ -8,12 +8,30 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    const checkBoolean = isIsogram('');
 
+    expect(typeof checkBoolean === 'boolean').toBeTruthy();
   });
```

Comment:
You can make it shorter, just 
`expect(typeof isIsogram('')).toBe('boolean)`

Reply:
You're right. Thanks